src/app/organizations/state/register-organization.service.ts

```diff
@@ -68,4 +155,43 @@ export class RegisterOrganizationService {
       });
     }
   }
+
+  /**
+   * Update the organization
+   * TODO: change URL when the organization name changes
+   *
+   * @param {FormsState['registerOrganization']} organizationFormState
+   * @param {number} organizationId
+   * @returns {void}
+   * @memberof RegisterOrganizationService
+   */
+  updateOrganization(organizationFormState: FormsState['registerOrganization'], organizationId: number): void {
+    if (!organizationFormState) {
+      console.error('Something has gone terribly wrong with the form manager');
+      return;
+    } else {
+      const editedOrganization: Organisation = {
+        name: organizationFormState.name,
+        topic: organizationFormState.topic,
+        link: organizationFormState.link,
+        location: organizationFormState.location,
+        email: organizationFormState.contactEmail,
+        // Setting approved to true to appease compiler.  Webservice should completely ignore this.
+        approved: true
+      };
+      this.alertService.start('Editing organization');
```

Comment:
I would say `Updating organization` -- Editing happens in the UI

Reply:
:ok_hand: